docs/en/OLED_Display.md

```diff
@@ -0,0 +1,199 @@
+For some more info about anchors check [Adafruit site](https://learn.adafruit.com/circuitpython-display-support-using-displayio/text). But keep in mind that KMK operates with `T`, `M`,`B` and `L`, `M`, `R` strings, not numbers.
+
+```python
+oled.entries = [
+    TextEntry(text="Layer = 1", x=128, y=0, x_anchor="R", y_anchor="T"), # text in Top Right corner
+    TextEntry(text="Macros", x=128, y=64, x_anchor="R", y_anchor="B"), # text in Bottom Right corner
+    TextEntry(text="Hey there!", x=64, y=32, x_anchor="M", y_anchor="M"), # text in the Middle of screen
+]
+keyboard.extensions.append(oled)
+```
+
+### Split
+As well as with images you can change displaying according to your layer or side of split keyboard.
+
+```python
+oled.entries = [
+    TextEntry(text="Longer text that", x=0, y=0, layer=0),
+    TextEntry(text="has been divided", x=0, y=12, layer=0, side="L"),
+    TextEntry(text="for an example", x=0, y=24, layer=0, side="R"),
+]
+keyboard.extensions.append(oled)
+```
+
+### Inverting
+inverts colours of your text. comes in handy for example as a good layer indicator.
+
+```python
+oled_ext = Oled(
+    entries=[
+        TextEntry(text='0 1 2 4', x=0, y=0),
+        TextEntry(text='0', x=0, y=0, inverted=True, layer=0),
+        TextEntry(text='1', x=12, y=0, inverted=True, layer=1),
+        TextEntry(text='2', x=24, y=0, inverted=True, layer=2),
+    ],
+)
+```
+
+# Fully working Example Code
+
+```python
+import board
+import busio
+from kmk.kmk_keyboard import KMKKeyboard
+from kmk.keys import KC
+from kmk.scanners import DiodeOrientation
+from kmk.modules.layers import Layers
+from kmk.extensions.oled import Oled, TextEntry, ImageEntry
+
+#EXTENSIONS
+keyboard = KMKKeyboard()
+Layers = Layers()
+keyboard.modules.append(Layers)
+
+
+#OLED
+keyboard.SDA = board.GP20
+keyboard.SCL = board.GP21
+i2c_bus = busio.I2C(board.GP21, board.GP20)
+oled = Oled(
+    entries=[
+        TextEntry(text='Layer: ', x=0, y=32, y_anchor='B'),
+        TextEntry(text='BASE', x=40, y=32, y_anchor='B', layer=0),
+        TextEntry(text='NUM', x=40, y=32, y_anchor='B', layer=1),
+        TextEntry(text='NAV', x=40, y=32, y_anchor='B', layer=2),
+        TextEntry(text='0 1 2', x=0, y=4),
+        TextEntry(text='0', x=0, y=4, inverted=True, layer=0),
+        TextEntry(text='1', x=12, y=4, inverted=True, layer=1),
+        TextEntry(text='2', x=24, y=4, inverted=True, layer=2),
+    ],
+    i2c=i2c_bus,
+    device_address=0x3C,
+    width=128,
+    height=64,
+    dim_time=10,
+    dim_target=0.1,
+    off_time=1200,
+    brightness=1,
+)
+
+keyboard.extensions.append(oled)
+```
+
+# Soldering
+
+Look online for your board pinout docs and solder accordingly to it.
+
+[Raspberry Pi Pico Pinouts](https://www.raspberrypi.com/documentation/microcontrollers/images/pico-pinout.svg)
+
+* GND - Ground Pin
+* VCC - 3V3 Pin
+* SCL - any SCL pin 
+* SDA - closest SDA pin
```

Comment:
Remove this. "Look it up online" isn't an actual helpfull statement related to this module, focusing on the pico layout for no reason doesn't make sense, and again a statement like "for SCL use an SCL pin" is not actually helpfull information.

Reply:
Sure thing. I added this because I had problems when soldering the display myself. I could not find information on where to solder these pins, and such information at that point would have been helpful to me. Just thought it might be useful for beginners.